lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -363,7 +367,7 @@ def _probabilities_to_percentiles(
             air_temperature at the required percentiles.
 
         """
-        threshold_coord = find_coordinate(forecast_probabilities, "threshold")
+        threshold_coord = forecast_probabilities.coord("threshold")
```

Comment:
Why do we need to change it now, we may just need to re-do  it later. But they are minor changes and you are getting rid of the find_coordinate function. I just hope we don't need to add it again later...